src/runtime/vdom/set-accessor.ts

```diff
@@ -170,3 +182,4 @@ export const setAccessor = (
 };
 const parseClassListRegex = /\s/;
 const parseClassList = (value: string | undefined | null): string[] => (!value ? [] : value.split(parseClassListRegex));
+const CAPTURE_EVENT_SUFFIX = 'Capture';
```

Comment:
Could we avoid the [re-instantiation of this regex above](https://github.com/ionic-team/stencil/pull/4968/files#diff-7f05c2f1645c35f29f698766956966cc2b5d915b37004316adfb635648ce9a62R120) on every call to by storing the regex as well?
```suggestion
const CAPTURE_EVENT_SUFFIX = 'Capture';
const CAPTURE_EVENT_REGEX = new RegExp(`${CAPTURE_EVENT_SUFFIX}$`);
```